### DISCUSSION TITLE
/data/bookie1/ledgers/current log clean

### DISCUSSION THREAD

**Discussion by zhbgs2000** [apache/pulsar#22574] [Q&A]
There are many .log files under the /data/bookie1/ledgers/current directory, which occupy a large amount of space. How can I clean them up?

![1713964216654](https://github.com/apache/pulsar/assets/19635652/9e257752-d22d-4f5c-9bd5-394dfba754f6)

**Comment by visortelle** [comment]
I didn't configure it for Bookkeeper, but I suppose that you can configure it in `log4j.properties` file:

> To enable logging for a bookie, create a log4j.properties file and point the BOOKIE_LOG_CONF environment variable to the configuration file.

https://bookkeeper.apache.org/docs/admin/bookies#logging

> RollingFileAppender does this. You just need to set maxBackupIndex to the highest value for the backup file.

https://stackoverflow.com/a/1050318/4182882
